### PR TITLE
Add HTC library

### PR DESCRIPTION
**Pull request by davideziviani**
From Pererira and Deschamps, IJR 82(2017), 325-334

### CODE REVIEW

File: PDSim/scroll/core.py [hunk 1]
```diff
@@ -1117,21 +1117,71 @@ def calculate_scroll_mass(self):
...
+            Heat transfer coefficient [kW/m2/K]
+            
+        Notes
+        -----
+        Pereira and Deschamps "A heat transfer correlation for the suction and compression chambers of scroll compressors" International Journal of Refrigeration, 82(2017), 325-334
+        """
+        
+        if hasattr(self,'HT_corr') == False and hasattr(self,'HTC') == False:
+            return 0.0
+        elif hasattr(self,'HTC') == True or self.HT_corr == 'HT-const':
```
Comment (ibell) [inline PDSim/scroll/core.py]: recommend just ``hasattr(self,'HTC')``

File: PDSim/scroll/core.py [hunk 2]
```diff
@@ -1117,21 +1117,71 @@ def calculate_scroll_mass(self):
...
+        ----------
+        hc : float
+            Heat transfer coefficient [kW/m2/K]
+            
+        Notes
+        -----
+        Pereira and Deschamps "A heat transfer correlation for the suction and compression chambers of scroll compressors" International Journal of Refrigeration, 82(2017), 325-334
+        """
+        
+        if hasattr(self,'HT_corr') == False and hasattr(self,'HTC') == False:
```
Comment (ibell) [inline PDSim/scroll/core.py]: More pythonic to do ``not hasattr(self, 'HT_corr')``

File: PDSim/scroll/core.py [hunk 3]
```diff
@@ -1571,7 +1619,7 @@ def mechanical_losses(self, shell_pressure = 'low:shell'):
             # Get the mean losses over one cycle
             self.losses.bearings  = np.trapz(self.losses.summed[_slice], theta)/theta_range
             
-            print('mechanical losses: ', self.losses.bearings)
+            print('mechanical losses [kW]: ', self.losses.bearings)
```
Comment (ibell) [inline PDSim/scroll/core.py]: This should go in a separate PR (one topic per PR), but I agree with it entirely!

**Comment by davideziviani** [comment]
I think I found another small issue with the if statements. I just tested a modified version that should work better.

**Comment by ibell** [comment]
Thanks, merging
